src/BaseTable.js

```diff
@@ -512,6 +512,7 @@ class BaseTable extends React.PureComponent {
           <ColumnResizer
             className={this._prefixClass('column-resizer')}
             column={column}
+            headerIndex={headerIndex}
```

Comment:
i know this is already merged but I was just curious to understand this multi resize.

because I dont see a prop to enable or disable this behavior?

Reply:
@dagadbm This property considered mandatory: https://github.com/datafold/react-base-table/blob/master/src/ColumnResizer.js#L241-L244